client/my-sites/plans/jetpack-plans/product-lightbox/constants.ts

```diff
@@ -24,8 +26,14 @@ export const PRODUCT_OPTIONS: Record< string, TranslateResult > = {
 	[ PRODUCT_JETPACK_SOCIAL_BASIC ]: translate( 'Social Basic', {
 		comment: 'Get up to 1000 shares',
 	} ),
+	[ PRODUCT_JETPACK_SOCIAL_BASIC_MONTHLY ]: translate( 'Social Basic', {
+		comment: 'Get up to 1000 shares',
+	} ),
 	[ PRODUCT_JETPACK_SOCIAL_ADVANCED ]: translate( 'Social Advanced', {
-		comment: 'Get unlimited shares',
+		comment: 'Get unlimited shares and share as a post.',
```

Comment:
I find this text somewhat confusing. Does Social Basic not allow sharing as a post? Does sharing as a post mean adding an image/video?

Reply:
>Does Social Basic not allow sharing as a post? Does sharing as a post mean adding an image/video?

That's correct, we allow sharing as post only on the advanced plan.. I have updated the text to describe that better, let me know what you think!